aea/cli/registry/search.py

```diff
@@ -0,0 +1,41 @@
+import click
+
+from utils import format_list, request_api
+
+
+@click.group()
+def search():
```

Comment:
Note, we have a cli search command implemented here: https://github.com/fetchai/agents-aea/blob/develop/aea/cli/search.py